src/Plugin.php

```diff
@@ -60,6 +63,17 @@ public static function init() {
     add_filter('woocommerce_thankyou_order_received_text', __NAMESPACE__ . '\GoogleTrustedStores::woocommerce_thankyou_order_received_text', 100, 2);
   }
 
+  /**
+   * Enqueues Trusted Shops product reviews related scripts.
+   *
+   * @implements wp_enqueue_scripts.
+   */
+  public static function enqueueProductReviewsScripts() {
+    if (is_product()) {
+      wp_enqueue_script(static::PREFIX, Plugin::getBaseUrl() . '/dist/scripts/product-reviews.min.js', ['jquery'], FALSE, TRUE);
```

Comment:
No cache for this? 🤔 

In addition, we generally use `script.js` as the name of the file.

Reply:
Also the name of the function looks weird to me. Why so specific? Can we not use a general `enqueueAssets` like in other plugins?